src/components/Pages/Service.js

```diff
@@ -50,11 +50,11 @@ const Service = ({
     <div className="wrapper wrapper--sm container-fluid">
       <PageHeader>{title}</PageHeader>
 
-      {steps &&
-        !!steps.length && (
+      {serviceSteps &&
+        !!serviceSteps.edges.length && (
```

Comment:
Can we remove edges when we process this data in static.config.js -- I think our components should be graphql neutral. We have other clean functions that set this pattern.